src/components/button-row/test-button-row.js

```diff
@@ -22,22 +22,4 @@ describe('ButtonRow', function() {
     const wrapper = renderComponent({buttons});
     expect(wrapper).toMatchSnapshot();
   });
```

Comment:
Another test should be added to check and make sure that it does indeed support multiple buttons.

Reply:
Yep, now added.